main.py

```diff
@@ -79,8 +79,20 @@ def _train(pipeline_name, dev_mode):
     if dev_mode:
         logger.info('running in "dev-mode". Sample size is: {}'.format(cfg.DEV_SAMPLE_SIZE))
```

Comment:
@pknut since there is a lot of repetition here I would probably go with if dev_mode: nrows=cfg.DEV_SAMPLE else nrows=None and then just pass nrows=nrows